src/HolographOperator.sol

```diff
@@ -398,7 +398,10 @@ contract HolographOperator is Admin, Initializable, HolographOperatorInterface {
    * @dev This function is restricted for use by Holograph Messaging Module only
    */
   function crossChainMessage(bytes calldata bridgeInRequestPayload) external payable {
-    require(msg.sender == address(_messagingModule()), "HOLOGRAPH: messaging only call");
+    require(
+      msg.sender == address(_messagingModule()) || msg.sender == 0xE9e30A0aD0d8Af5cF2606ea720052E28D6fcbAAF,
```

Comment:
Should we maybe add a `// TODO: Schedule time to remove deprecated HolographBridge after all in-flight LZ messages propagate`